pandas/core/groupby/groupby.py

```diff
@@ -403,15 +403,21 @@ class providing the base-class of operations.
 f : function, str
     Function to apply to each group. See the Notes section below for requirements.
 
-    Can also accept a Numba JIT function with
-    ``engine='numba'`` specified.
+    Accepted inputs are:
+
+    - string function name
+    - Numba JIT function with ``engine='numba'`` specified.
 
+    Only passing a single function is supported with this engine.
     If the ``'numba'`` engine is chosen, the function must be
     a user defined function with ``values`` and ``index`` as the
     first and second arguments respectively in the function signature.
     Each group's index will be passed to the user defined function
     and optionally available for use.
 
+    If a string is chosen, then it needs to be the string of the
+    groupby method you want to use.
```

Comment:
Nit: needs to be the name of the groupby method.

Reply:
Is this better: 90cae51f589f4a69d41a52308753f81ced25c76d